### PR TITLE
Fix CI by specifying valid, specific simulators

### PR DESCRIPTION
**Pull request by mdiep**

### CODE REVIEW

**Comment by mdiep** [comment]
(The macOS build still fail because there's an actual failing test atm.)

**Comment by andersio** [comment]
The failing test was from #3433. The fix is ready in #3435.

**Comment by mdiep** [comment]
Yup. That's on my list. Just trying to get iOS and watchOS passing here.

File: .travis.yml [hunk 1]
```diff
@@ -40,7 +39,7 @@ matrix:
       env:
         - XCODE_SDK=watchsimulator
         - XCODE_ACTION=build
-        - XCODE_DESTINATION="platform=watchOS Simulator,name=Apple Watch - 38mm"
+        - XCODE_DESTINATION="platform=watchOS Simulator,name=Apple Watch - 38mm,OS=2.2"
```
Comment (ikesyo) [inline .travis.yml]: It's surprising that there are duplicated simulators of watchOS 3.2 in this time 😲

Comment (mdiep) [inline .travis.yml]: 🤷🏼‍♂️

Comment (ikesyo) [inline .travis.yml]: Filed https://github.com/travis-ci/travis-ci/issues/7580

**Pull request merged**